### DISCUSSION TITLE
\n not accepted by watchtower in docker container

### DISCUSSION THREAD

**Discussion by To0wnn** [containrrr/watchtower#1633] [Q&A] [answered]
### Describe the bug

\n not accepted by watchtower in docker container

Getting: ERRO[0000] Could not use configured notification template: template: :19: function "n" not defined. Using default template

### Steps to reproduce

run the docker container

![image](https://user-images.githubusercontent.com/25762347/233776440-8d0f6976-71ca-40d3-8d5d-4b54591db80e.png)

### Expected behavior

not getting the error

### Environment

Docker version 20.10.3, build 55f0773

### Your logs

```text
ERRO[0000] Could not use configured notification template: template: :19: function "n" not defined. Using default template
```

**Comment by piksel** [comment, marked as answer]
Look at your syntax highlighting, you are using " to quote the value in your shell, and then you are using the same quote to create the string (effectively ending your variable quote). Either use \`\n\`, a literal newline or `{{ println }}`

**Comment by To0wnn** [comment]
`{{ println }}` is working! Thnx!
